api/edge/v1alpha1/types.go

```diff
@@ -205,8 +205,9 @@ type PlacementDecisionSpec struct {
 }
 
 // DownsyncObjectReferences explicitly defines the objects to be down-synced.
-// The ClusterScope list defines the cluster-scope objects, while NamespacedObjects packs individual objects
-// identifiable by namespace & name.
+// The ClusterScope list defines the cluster-scope objects, NamespaceScope list
+// defines the namespace-scope objects and generation represents the internal spec
```

Comment:
Maybe revision number?  version ? sequence?  personally I find generation confusing

Reply:
I agree that generation might get confusing. same probably goes for version which might be confusing with ResourceVersion.
revision sounds good to me.
if we can get an agreement on the general direction with @pdettori and @MikeSpreitzer I'm happy to rename the field to `revision`.